packages/manager/modules/pci/src/projects/project/kubernetes/details/service/translations/Messages_fr_FR.json

```diff
@@ -17,8 +17,9 @@
   "kube_service_cluster_status_UPDATING": "Mise à jour",
   "kube_service_cluster_status_DELETING": "En cours de suppression",
   "kube_service_cluster_status_RESETTING": "En cours de réinitialisation",
-  "kube_service_cluster_version": "Version de Kubernetes installée",
-  "kube_service_cluster_update_available": "Une mise à jour mineure est disponible",
+  "kube_service_cluster_version": "Version mineure de Kubernetes",
+  "kube_service_cluster_update_available": "Une nouvelle mise à jour touchant à la sécurité (<span class='font-italic'>patch version</span>) est disponible",
```

Comment:
If not possible to split the actual sentence, please consider using as less as HTML/class as possible. I think using a `<i>` tag could be enough.